tests/std/tests/Dev11_0836436_get_time/test.cpp

```diff
@@ -792,16 +794,17 @@ void test_invalid_argument() {
     time_t t = time(nullptr);
     tm currentTime;
     localtime_s(&currentTime, &t);
+    currentTime.tm_hour = 25; // set invalid hour

```

Comment:
Windows implementation of `strftime()` does crash when invalid `struct tm` is supplied. However, in `std::put_time()` shouldn't that be just `ios_base::badbit` instead of a crash?

Reply:
The Standard's wording is less clear than I'd like (probably because it's from an early era when specifications were written less precisely), but it seems to me that crashing is disallowed, because "nothing permits it to fail". I suspect that `failbit` is actually what should be set, since the requested conversion can't be performed due to bogus input. `badbit` generally means "something is wrong with the stream itself".

A followup PR would be reasonable.